cabal-testsuite/PackageTests/Check/PackageFiles/VersionBounds/cabal.out

```diff
@@ -0,0 +1,18 @@
+# cabal check
+These warnings may cause trouble when distributing the package:
+Warning: [missing-upper-bounds] On library, these packages miss upper bounds:
+  - missing-upper
+  - exclusive-minimums-missing-upper
+  - or-exclusive-minimums-missing-upper
+  - or-inclusive-maximums-missing-upper
+Please add them. There is more information at https://pvp.haskell.org/
+Warning: [le-upper-bounds] On library, these packages have less than or equals (<=) upper bounds:
+  - inclusive-maximums
+  - and-inclusive-maximums
+  - or-inclusive-maximums-missing-upper
+Please use less than (<) for upper bounds.
+Warning: [gt-lower-bounds] On library, these packages have greater than (>) lower bounds:
+  - exclusive-minimums-missing-upper
+  - and-exclusive-minimums
+  - or-exclusive-minimums-missing-upper
+Please use greater than or equals (>=) for lower bounds. 
```

Comment:
On command line git, I see an extra ␣ on the last line.

Reply:
@ffaf1 are you still seeing this? Is that the same thing as `make fix-whitespace` corrects?

Reply:
There is still an extra space in the last line, you can see it here.

https://github.com/cabalism/cabal/blob/fix/check-version-bounds/cabal-testsuite/PackageTests/Check/PackageFiles/VersionBounds/cabal.out

Reply:
I see it now. Rerunning the test with `--accept` doesn't rewrite it when there is only a trailing whitespace differerence. I made a non-whitespace edit ("PLEASE") to the `cabal.out` file and this time the output was regenerated:

```
$ cabal run cabal-testsuite:cabal-tests -- --with-cabal=./dist-newstyle/build/x86_64-linux/ghc-9.10.1/cabal-install-3.15.0.0/x/cabal/build/cabal/cabal cabal-testsuite/PackageTests/Check/PackageFiles/VersionBounds/cabal.test.hs --accept
...
--- /tmp/cabal-testsuite-92156/cabal.dist/cabal.out.normalized	2024-12-29 16:59:13.253824459 -0500
+++ /tmp/cabal-testsuite-92156/cabal.dist/cabal.comp.out.normalized	2024-12-29 16:59:13.253824459 -0500
@@ -15,4 +15,4 @@
   - exclusive-minimums-missing-upper
   - and-exclusive-minimums
   - or-exclusive-minimums-missing-upper
-PLEASE use greater than or equals (>=) for lower bounds. 
+Please use greater than or equals (>=) for lower bounds.
```